goshopify.go

```diff
@@ -32,6 +32,9 @@ const (
 var (
 	// version regex match
 	apiVersionRegex = regexp.MustCompile(`^[0-9]{4}-[0-9]{2}$`)
+
+	// MaxLoggedHTTPBodyBytes - limits the memory usage per logged request or response body to 32KiB
+	MaxLoggedHTTPBodyBytes int64 = 32000 // ~32KiB default to something more than reasonable, but expose it for configuration
```

Comment:
package var works but could be more useful to config per client. i needed flexibility for multi clients and updated a bunch of things to accommodate that so I'd like to at least advocate for that concept. perhaps adding another option to set this like WithLogger and storing it on the client struct?